content/pages/changelog/changelog-2018.md

```diff
@@ -0,0 +1,335 @@
+---
+alias_paths:
+  - /changelog/changelog-2018
+---
+
+# Changelog (2018)
+
+For current changelog entries, please see
+[2019 Release Notes](pages://changelog/).
+
+## 2018-12-21 Box CLI V2 Released
+
+A major version release (v2) has been made to the [Box CLI](guide://tooling/sdks/cli).
+See the following links for updated information:
+
+* [Release notes](cli_v2_release_notes):
```

Comment:
The syntax for a reference link in markdown is `[Text][ref]`. Not that all braces are square braces. Normal links follow the `[Text](url)` syntax where the second braces are round ones.